makiflow/trainers/nn_render/render_trainer.py

```diff
@@ -287,9 +387,9 @@ def _perform_testing(self, exp_params, save_path, path_to_weights, FPS=25):
         for i in range(len(origin_image)):
             mask = masks[i]
             image = copy.deepcopy(origin_image[i])
-            image[:, :, 0] *= (mask != 1)
-            image[:, :, 1] *= (mask != 1)
-            image[:, :, 2] *= (mask != 1)
+            image[..., 0] *= (mask != 1)
+            image[..., 1] *= (mask != 1)
+            image[..., 2] *= (mask != 1)
             without_face.append(image.astype(np.float32))
 
         batch_size = exp_params[ExpField.BATCH_SIZE]
```

Comment:
Why use `exp_params[ExpField.BATCH_SIZE]` instead of 1? If you use training batch size, some of the uv maps at the end will be skipped. It just doesn't make sense to do it this way.